### PR TITLE
update kfctl doc for kustomize

### PR DESCRIPTION
**Pull request by lluunn**
related to #864 . 

/cc @jlewi  @kkasravi

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/887)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-887--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/deploy-cli/

File: content/docs/gke/deploy/deploy-cli.md [hunk 1]
```diff
@@ -151,29 +151,25 @@ following:
...
-* **${KFAPP}/k8s_specs** is a directory that contains YAML specifications
-  for some daemons deployed on your Kubernetes Engine cluster.
-
-* **${KFAPP}/ks_app** is a directory that contains the 
-  [ksonnet](https://ksonnet.io) application for Kubeflow.
+* **kustomize** is a directory that contains the kustomize packages for Kubeflow applications.
 
   * The directory is created when you run `kfctl generate`.
-  * You can use ksonnet to customize Kubeflow.
+  * You can customize the k8s resouces (modify the manifests and run `kfctl apply` again)
```
Comment (sarahmaddox) [inline content/docs/gke/deploy/deploy-cli.md]: Fix a typo and an abbreviation:

s/k8s resouces/Kubernetes resources

Comment (sarahmaddox) [inline content/docs/gke/deploy/deploy-cli.md]: Add a full stop (period) at the end of the sentence.

**Comment by lluunn** [comment]
fixed, thanks!

**Pull request merged**